src/ActiveLogin.Authentication.BankId.AspNetCore/BankIdAuthenticationHandler.cs

```diff
@@ -166,6 +167,23 @@ bool TryGetPinString(out string s)
             return null;
         }
 
+        private string GetCancelReturnUrl(AuthenticationProperties properties)
+        {
+            // Default to root if no return url is set
+            var cancelReturnUrl = properties.Items["cancelReturnUrl"] ??
```

Comment:
The test is failing here now. I think you need to check if the properties contains the value.

Reply:
Doh.